integration-tests/src/test/java/oracle/kubernetes/operator/ItOperatorUpgrade.java

```diff
@@ -238,41 +141,74 @@ public void testOperatorUpgradeFrom2_3_0() throws Exception {
    * @param restart boolean parameter used to determine if a restart of domain is checked
    * @throws Exception when upgrade fails or basic usecase testing or scaling fails.
    */
-  private void upgradeOperator(boolean restart) throws Exception {
+  private void upgradeOperator() throws Exception {
     operator.callHelmUpgrade("image=" + OP_TARGET_RELEASE);
-    if (restart) {
-      checkDomainRollingRestarted();
-    }
-    checkOperatorVersion();
+    checkDomainNotRestarted();
+    checkCrdVersion();
+    // checkDomainApiVersion(); Not Needed 
     testClusterScaling(operator, domain, false);
   }
 
   /**
-   * checks the expected version of the upgraded operator in a loop. In Jenkins it takes nearly 8
-   * minutes to show the updated value of the domain CRD.
+   * Checks Expected/Upgraded ApiVersion of the Domain Object in a loop. 
+   * In Jenkins it takes nearly 8 minutes to show the updated value 
+   * 
+   * The .apiVersion of the domain will not be updated until the domain is 
+   * written again (e.g. by the operator when updating status)
+   * This contributes to long time period. We can skip this check 
    *
    * @throws Exception when version does not match
    */
-  private void checkOperatorVersion() throws Exception {
+  private void checkDomainApiVersion() throws Exception {
     boolean result = false;
     LoggerHelper.getLocal().log(
         Level.INFO,
         "Checking for the domain apiVersion "
             + getDomainApiVersion()
             + " in a loop for up to 15 minutes");
-    for (int i = 0; i < 900; i = i + 10) {
+    for (int i = 0; i < 120; i++) {
       ExecResult exec =
           TestUtils.exec(
               "kubectl get domain -n " + DOM_NS + "  " + DUID + " -o jsonpath={.apiVersion}", true);
       if (exec.stdout().contains(getDomainApiVersion())) {
-        LoggerHelper.getLocal().log(Level.INFO, "Got the expected apiVersion");
+        int ttime = 10 * (i - 1); 
+        LoggerHelper.getLocal().log(Level.INFO, "Got the expected API Version after [" + ttime + "] Seconds");
+        result = true;
+        break;
+      }
+      Thread.sleep(1000 * 10);
```

Comment:
i could be wrong, but i think a `Thread.sleep()` needs to be in a `try { .. } catch (InterruptedException) {} `
@rjeberhard ?

Reply:
Added InterruptedException to the Thread.sleep
 try {
        Thread.sleep(BaseTest.getWaitTimePod() * 1000);
      } catch (InterruptedException e) {
        System.out.println("Got InterruptedException " + e);
      }